runelite-client/src/main/java/net/runelite/client/callback/Hooks.java

```diff
@@ -367,19 +365,6 @@ public static boolean menuActionHook(int actionParam, int widgetId, int menuActi
 		return menuOptionClicked.isConsumed();
 	}
 
-	public static void addChatMessage(int type, String name, String message, String sender)
-	{
-		if (log.isDebugEnabled())
```

Comment:
Any reason this is removed? The Hooks logger is accessible. We could also maybe just inject a logger.